lib/models/project.js

```diff
@@ -18,6 +18,7 @@ const PackageInfoCache = require('./package-info-cache');
 const PerBundleAddonCache = require('./per-bundle-addon-cache');
 
 const instantiateAddons = require('./instantiate-addons');
+const { HostInfoCache } = require('./host-info-cache');
```

Comment:
Given that all the other modules here are just the class, not a class inside an object, and there is nothing else in this particular object, would it be better to just have `const HostInfoCache` w/o the {} wrapper? More consistent at least, if not any better.